symengine/visitor.h

```diff
@@ -209,8 +209,8 @@ class CoeffVisitor : public StopVisitor<CoeffVisitor> {
     }
 };
 
-RCP<const Basic> coeff(const Basic &b, const RCP<const Symbol> &x,
-        const RCP<const Integer> &n);
+RCP<const Basic> coef(const Basic &b, const RCP<const Symbol> &x,
+                      const RCP<const Integer> &n);
```

Comment:
Not yet

On Sun, Jul 12, 2015 at 8:45 PM, Ondřej Čertík <notifications@github.com>
wrote:

> In symengine/visitor.h
> <https://github.com/sympy/symengine/pull/505#discussion_r34422568>:
>
> > @@ -209,8 +209,8 @@ class CoeffVisitor : public StopVisitor<CoeffVisitor> {
> >      }
> >  };
> >
> > -RCP<const Basic> coeff(const Basic &b, const RCP<const Symbol> &x,
> > -        const RCP<const Integer> &n);
> > +RCP<const Basic> coef(const Basic &b, const RCP<const Symbol> &x,
> > +                      const RCP<const Integer> &n);
>
> Did you find a way to avoid this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/505/files#r34422568>.
>